packages/forc-bin/src/install.ts

```diff
@@ -1,27 +1,46 @@
 #!/usr/bin/env node
 
-import fs from 'fs/promises';
+import { execSync } from 'child_process';
+import { existsSync, rmSync, writeFileSync } from 'fs';
 import fetch from 'node-fetch';
-import path from 'path';
+import { join } from 'path';
 import sh from 'shelljs';
 
 import { getCurrentVersion, getPkgPlatform } from './shared';
 
 (async () => {
   const pkgPlatform = getPkgPlatform();
   const forcVersion = await getCurrentVersion();
+
   const pkgName = `forc-binaries-${pkgPlatform}.tar.gz`;
   const pkgUrl = `https://github.com/FuelLabs/sway/releases/download/v${forcVersion}/${pkgName}`;
-  const pkgPath = path.join(__dirname, pkgName);
-  const binDir = path.join(__dirname, '../');
 
-  // Download
-  const buf = await fetch(pkgUrl).then((r) => r.buffer());
-  await fs.writeFile(pkgPath, buf);
+  const pkgPath = join(__dirname, pkgName);
+  const binDir = join(__dirname, '../');
+
+  const binPath = join(binDir, 'forc-binaries', 'forc');
+  let versionMatches = false;
+
+  if (existsSync(binPath)) {
+    const binRawVersion = execSync(`${binPath} --version`).toString().trim();
+    const binVersion = binRawVersion.match(/([.0-9]+)/)?.[0];
+
+    versionMatches = binVersion === forcVersion;
+    console.info({ expected: forcVersion, received: binVersion });
```

Comment:
Just missing a comment to ignore console warning

Reply:
Should I ignore it with a comment or use another method for logging info to the console?

Reply:
Just ignore with the comment